cli/cli.go

```diff
@@ -106,6 +124,14 @@ Version:
 	rootCmd.PersistentFlags().BoolVar(&cfg.Logger.DisableColor, "no-color", false, "disable colors in command output [$NO_COLOR=1]")
 	rootCmd.PersistentFlags().StringSlice("pragma", nil, "database pragma key-value pairs (e.g. --pragma mmap_size=536870912 --pragma synchronous=OFF)")
 
+	if locker.Locked() {
+		rootCmd.PersistentFlags().StringVar(&cfg.Locker.MachineFilePath, "locker-machine-file-path", try.Try(try.Env("RELAY_LOCKER_MACHINE_FILE_PATH"), try.Static("./relay.lic")), "the path to a machine file for unlocking relay [$RELAY_LOCKER_MACHINE_FILE_PATH=./relay.lic]")
+		rootCmd.PersistentFlags().StringVar(&cfg.Locker.LicenseKey, "locker-license-key", try.Try(try.Env("RELAY_LOCKER_LICENSE_KEY"), try.Static("")), "the path to a license key for unlocking relay [$RELAY_LOCKER_LICENSE_KEY=xxx]")
+
+		_ = rootCmd.MarkPersistentFlagRequired("locker-machine-file-path")
```

Comment:
These aren't working for some reason. Maybe because we're returning an error in pre-run?